pkg/logs/input/docker/tailer.go

```diff
@@ -187,10 +187,12 @@ func (t *Tailer) readForever() {
 				case err == io.EOF:
 					// This error is raised when the container is stopping
 					// or when the container has not started to output logs yet.
```

Comment:
we probably want to add a line about the file rotation that can happen when the logging drive is file